qiskit_aer/primitives/estimator.py

```diff
@@ -344,7 +345,11 @@ def _compute_with_approximation(
         self, circuits, observables, parameter_values, run_options, seed
     ):
         if self._abelian_grouping:
-            raise AerError("approximation and abelian_grouping cannot be True at the same time.")
+            warn(
```

Comment:
Maybe an alternative would be to state in the parameter docstring for abelian_grouping that this setting is ignored when approximation is True. Abelian grouping is True by default (I cannot see most people wanting to disable grouping) hence if you just turn approximation True you will get this warning so it maybe annoying to people to have to go set grouping to False to avoid the warning.